### PR TITLE
data:jainamshah3638

### PR DESCRIPTION
**Pull request by jainamshah3638**
Jainam shah Profile added 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3466"></a>

### CODE REVIEW

File: data/jainamshah3638.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   "socials": [
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/jainam-shah-21284814b/"},
+    { "icon": "FaTwitter", "url": "https://twitter.com/Jainamh_shah" },
+    { "icon": "FaGithub", "url": "https://github.com/jainamshah3638" }
+  ],
+  "avatar": "https://github.com/jainamshah3638.png",
+    "links": [
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://twitter.com/EddieAndSaraExplore",
```
Comment (EmmaDawsonDev) [inline data/jainamshah3638.json:16]: Please update to your own account

File: data/jainamshah3638.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
```
Comment (EmmaDawsonDev) [inline data/jainamshah3638.json:21]: Please update this link too

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Just a couple of links to update then this should be good to merge.

File: data/jainamshah3638.json (hunk 1 above)
Comment (jainamshah3638) [reply data/jainamshah3638.json]: Changes done as you told 
Q

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**